packages/web-comment/src/routes/index.js

```diff
@@ -0,0 +1,38 @@
+const { readdirSync } = require('fs');
+
+// works line Next, urls come from dir names in routes director
+
+const dirNames = readdirSync(__dirname).filter((name) => /.*(?<!(\.js))$/.exec(name));
```

Comment:
Is it better to use `lstatSync` and `isDirectory()`? https://nodejs.org/docs/latest-v18.x/api/fs.html#fslstatsyncpath-options
Otherwise we have to update this regex if we add other file types to this directory.

Reply:
🧠